src/pages/Trade/StandaloneTrade.tsx

```diff
@@ -0,0 +1,88 @@
+import { useCallback, useRef } from 'react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { MemoryRouter, Route, Switch, useHistory, useLocation } from 'react-router-dom'
+
+import { LimitOrder } from '@/components/MultiHopTrade/components/LimitOrder/LimitOrder'
+import { LimitOrderRoutePaths } from '@/components/MultiHopTrade/components/LimitOrder/types'
+import { Claim } from '@/components/MultiHopTrade/components/TradeInput/components/Claim/Claim'
+import { ClaimRoutePaths } from '@/components/MultiHopTrade/components/TradeInput/components/Claim/types'
+import type { TradeCardProps } from '@/components/MultiHopTrade/MultiHopTrade'
+import { MultiHopTrade } from '@/components/MultiHopTrade/MultiHopTrade'
+import { TradeInputTab, TradeRoutePaths } from '@/components/MultiHopTrade/types'
+
+type StandaloneTradeProps = Omit<TradeCardProps, 'onChangeTab'>
+
+// i.e all the routes that Trade.tsx handles
+const initialEntries = [
+  TradeRoutePaths.Input,
+  TradeRoutePaths.Confirm,
+  TradeRoutePaths.VerifyAddresses,
+  TradeRoutePaths.QuoteList,
+  TradeRoutePaths.Quotes,
+
+  LimitOrderRoutePaths.Input,
+  LimitOrderRoutePaths.Confirm,
+  LimitOrderRoutePaths.AllowanceApproval,
+  LimitOrderRoutePaths.PlaceOrder,
+  LimitOrderRoutePaths.Orders,
+
+  ClaimRoutePaths.Select,
+  ClaimRoutePaths.Confirm,
+  ClaimRoutePaths.Status,
+]
+
+// A standalone version of the trade page routing as an HOC, without it being a page, and without leveraging top-level routes but rather
+// wrapping trade routes with a MemoryRouter.
+// This allows us to keep the logic the same as usual and don't add any more refactor to https://github.com/shapeshift/web/pull/8983
+// We should prefer regular routing whenever possible or at least use `MemoryRouter`s with caution, but for the time being, this is a pragmatic solution
+// to allow us to have our cake and eat it
```

Comment:
May have spoken too fast re: removing `<MemoryRouter />` but hey, having our cake (no `<MemoryRouter />` on the page version of trade, which was causing issues) and eating it (swapper still working as standalone i.e without being in the context of /trade, /limit or /claim pages) doesn't sound too bad?

Reply:
Seems acceptable, reduce the refactoring a lot, without this we would need to kick all the router logic from the inside of every swapper bits, which would be  both very risky and I'm not even sure the homemade solution would be sexier than a MemoryRouter